### PR TITLE
Bugfix/issue 246

### PR DESCRIPTION
**Pull request by rambrus**
*Issue #, if available:*
https://github.com/awslabs/deequ/issues/246

*Description of changes:*
@sscdotopen this PR is fixing the Histogram metrics backward compatibility issue, also extended the tests to cover this edge case.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by rambrus** [comment]
@sscdotopen could you please take a look at this bugfix?

File: src/main/scala/com/amazon/deequ/analyzers/runners/AnalyzerContext.scala [hunk 1]
```diff
@@ -87,7 +87,11 @@ object AnalyzerContext {
       .filter({ case (_, metrics) => metrics.value.isSuccess })
       // Get metrics as Double and replace simple name with description
       .flatMap({ case (analyzer, metrics) =>
-        metrics.flatten().map(renameMetric(_, describeAnalyzer(analyzer))) })
+        metrics.flatten().map(metric => {
```
Comment (tdhd) [inline src/main/scala/com/amazon/deequ/analyzers/runners/AnalyzerContext.scala:90]:
```suggestion
        metrics.flatten().map(metric =>
```

File: src/main/scala/com/amazon/deequ/analyzers/runners/AnalyzerContext.scala [hunk 2]
```diff
@@ -87,7 +87,11 @@ object AnalyzerContext {
...
+          val description = describeMetric(analyzer, metric)
+          renameMetric(metric, description)
+        })
```
Comment (tdhd) [inline src/main/scala/com/amazon/deequ/analyzers/runners/AnalyzerContext.scala:93]:
```suggestion
        )
```

**Comment by rambrus** [comment]
@tdhd I got a notification e-mail about your comment, but can't see it in github. Can you please double-check / resend?

**Comment by rambrus** [comment]
@sscdotopen have you had a chance to take a look at this bugfix?

**Comment by tdhd** [comment]
@rambrus I am having a look now

**Comment by tdhd** [comment]
LGTM, the PR is merged now.

**Comment by rambrus** [comment]
@tdhd: thanks for checking, would it be possible to publish a new RC build that has this bugfix?
